pkg/cvo/sync_worker.go

```diff
@@ -1096,6 +1099,25 @@ func (r *consistentReporter) ContextError(err error) error {
 	return errContext{fmt.Errorf("update %s at %d of %d", err, r.done, r.total)}
 }
 
+func (r *consistentReporter) CompleteWithErrors(errs []error) {
```

Comment:
nit:  this is no longer used with the conditional `Errors` call, so we can drop it.